src/style.css

```diff
@@ -1,3 +1,80 @@
+html {
+  --main-color: #fff;
+  --for-active-links: #00acdc;
+  --for-links: #000;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: var(--main-color);
+}
+
+.nav {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.logo__image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  position: relative;
+  color: var(--for-links);
+  padding: 22px 0;
```

Comment:
```suggestion
  padding: 23px 0;
```
<img width="910" alt="Screenshot 2023-08-24 at 09 11 35" src="https://github.com/mate-academy/layout_moyo-header/assets/53228259/eddb0efa-1e45-4a7b-a007-4602c0cdc57a">
<img width="1230" alt="Screenshot 2023-08-24 at 09 11 48" src="https://github.com/mate-academy/layout_moyo-header/assets/53228259/4a3c791f-3621-4afe-8be8-4d13f6dd8317">

Reply:
i dont understand what the difference?and why its so important?